### DISCUSSION TITLE
自駕巴士來了！新北「智駕巴士」最快三月後上路｜智慧城市懶人包#76

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#713] [General]
https://futurecity.cw.com.tw/article/1184
　太狠了，阿公阿罵都說讚，不用再看司機的臭臉

**Comment by yjpxi** [comment]
自駕巴士可以提供固定班次和定點服務，可以讓公共交通覆蓋更全面，尤其是在交通服務缺乏的地區可以提高居民的出行便利性，增加他們的出行選擇。
